digdag-tests/src/test/resources/acceptance/td/td_ddl/rename_not_exists.dig

```diff
@@ -0,0 +1,10 @@
+timezone: UTC
+
++rename:
```

Comment:
As this workflow contains two tasks with the same name I guess the acceptance test is failing due to that, not due to the `rename_tables` error.

Indeed, running the test produces this output.

```
...
error: Validating project failed
workflow /var/folders/07/xpg6gk057yx389cs5pgrwx5w0000gq/T/junit7289696207733610814/junit3204514592641809089/rename_not_exists.dig Duplicated keys: [+rename] (model validation)
```